src/mapbox/mapbox.js

```diff
@@ -346,6 +350,7 @@ export default class Mapbox {
 
     // Creation only props
     this.mapboxgl.accessToken = props.mapboxApiAccessToken || defaultProps.mapboxApiAccessToken;
+    this.mapboxgl.config.API_URL = props.mapboxApiUrl;
```

Comment:
there is another prop `baseApiUrl` documented in [mapbox-gl-js](https://docs.mapbox.com/mapbox-gl-js/api/#baseapiurl), do you know the difference between `baseApiUrl` and `this.mapboxgl.config.API_URL`?

Reply:
@xintongxia you are right, that method has been available since v0.52. I'll update it.